### PR TITLE
Update values impact

### PR DESCRIPTION
**Pull request by lscheske**
* Update index
* Add California vision 2023 to policy-documents.md

-----
[View rendered activities/value-and-impact/index.md](https://github.com/publiccodenet/about/blob/update-values-impact/activities/value-and-impact/index.md)
[View rendered activities/value-and-impact/policy-documents.md](https://github.com/publiccodenet/about/blob/update-values-impact/activities/value-and-impact/policy-documents.md)

### CODE REVIEW

File: activities/value-and-impact/index.md [hunk 1]
```diff
@@ -4,9 +4,14 @@ type: Index
 
 # Value and impact
 
-These are all the activities, documents and templates for value and impact:
+These are activities, documents and evidence for how we create value and impact:
```
Comment (ElenaFdR) [inline activities/value-and-impact/index.md:7]: I *love* this.

File: activities/value-and-impact/policy-documents.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ explains: where we are mentioned in public policy documents
 
 # Policy documents that mention the Foundation for Public Code
 
+## 2021
+
+* State of California: [Vision 2023 - California Technology Strategic Plan](https://vision2023.cdt.ca.gov/pdf/Vision-2023-California-Technology-Strategic-Plan.pdf)
```
Comment (ElenaFdR) [inline activities/value-and-impact/policy-documents.md:10]:
```suggestion
* State of California: [Vision 2023 - California Technology Strategic Plan](https://vision2023.cdt.ca.gov/) [[PDF](https://vision2023.cdt.ca.gov/pdf/Vision-2023-California-Technology-Strategic-Plan.pdf)
```

**Review by ElenaFdR — COMMENTED** [review]
Looks good! It's generally better to link to a webpage hosting a PDF than directly to a PDF so that a user can decide if they want to download the PDF (for example if someone's on low bandwidth, or a mobile).
This is especially important with documents that get updated (so that users are always sent to the most recent version of the PDF, or at least to a page that tells them that this strategy is cancelled or obsolete - assuming the receiving website is that well maintained).

**Comment by lscheske** [comment]
> Looks good! It's generally better to link to a webpage hosting a PDF than directly to a PDF so that a user can decide if they want to download the PDF (for example if someone's on low bandwidth, or a mobile).
> This is especially important with documents that get updated (so that users are always sent to the most recent version of the PDF, or at least to a page that tells them that this strategy is cancelled or obsolete - assuming the receiving website is that well maintained).

Ok! I was wondering about exactly this yesterday! I * think * I can make that change for most of them, I'll go back and do that now.

File: activities/value-and-impact/index.md [hunk 3]
```diff
@@ -4,9 +4,14 @@ type: Index
...
+
+## Value
 
 * [User mapping and value proposition hypotheses](user-mapping/index.md) for potential users of Foundation for Public Code products and activities
 * [Existing pain points](pain-points/index.md) for users of the Foundation for Public Code products and activities
+
+## Impact
+
 * [How public code supports the Sustainable Development Goals](sustainable-development-goals.md)
```
Comment (ElenaFdR) [inline activities/value-and-impact/index.md:16]: I also love this!

Comment (lscheske) [reply activities/value-and-impact/index.md:16]: can this be approved? Then I can merge :)

File: activities/value-and-impact/policy-documents.md [hunk 4]
```diff
@@ -5,12 +5,16 @@ explains: where we are mentioned in public policy documents
...
+## 2021
+
+* State of California: [Vision 2023 - California Technology Strategic Plan](https://vision2023.cdt.ca.gov/) [[PDF]](https://vision2023.cdt.ca.gov/pdf/Vision-2023-California-Technology-Strategic-Plan.pdf)
+
 ## 2020
 
 * Netherlands national government: [Letter to parliament about the release of government software source code](https://www.rijksoverheid.nl/documenten/kamerstukken/2020/04/17/kamerbrief-inzake-vrijgeven-broncode-overheidssoftware) (in Dutch)
 * Province of South Holland: [Council decision on membership of the Foundation for Public Code](https://www.zuid-holland.nl/overons/bestuur-zh/gedeputeerde-staten/besluiten/2020/september/1-september/lidmaatschap-foundation-for-public-code-vereniging/) (in Dutch)
-* Amsterdam municipality: [Mid-term report of the A Digital City Agenda (Agenda Digitale Stad)](https://amsterdam.raadsinformatie.nl/document/9725293/1/09012f9783374971) (in Dutch)
+* Amsterdam municipality: [Mid-term report of the A Digital City Agenda (Agenda Digitale Stad)[PDF]](https://amsterdam.raadsinformatie.nl/document/9725293/1/09012f9783374971) (in Dutch)
```
Comment (Ainali) [inline activities/value-and-impact/policy-documents.md:16]:
```suggestion
* Amsterdam municipality: [Mid-term report of the A Digital City Agenda (Agenda Digitale Stad) [PDF]](https://amsterdam.raadsinformatie.nl/document/9725293/1/09012f9783374971) (in Dutch)
```

Comment (Ainali) [reply activities/value-and-impact/policy-documents.md:16]: Missing space here (we have that before the other PDFs).

**Review by Ainali — CHANGES_REQUESTED** [review]

**Review by Ainali — APPROVED** [review]
Good to go!

**Pull request merged**